src/typings.ts

```diff
@@ -415,6 +415,13 @@ export interface DelegationFeesRaw {
   currencyNetworkOfFees?: string
 }
 
+export interface PaidDelegationFeesRaw {
+  feeSender: string
+  feeRecipient: string
+  totalFee: string
+  currencyNetworkOfFees: string
```

Comment:
The keys appear unnecessary long to me. This is the interface for the ...**Fees**... So I don't think the key names need that as well.

```suggestion
  sender: string
  recipient: string
  totalValue: string
  currencyNetwork: string
```

Reply:
They do need that as they are the keys of the object returned by the relay endpoint.